mysos/scheduler/scheduler.py

```diff
@@ -425,6 +469,18 @@ def statusUpdate(self, driver, status):
         log.error("Status update failed due to launcher error: %s" % e.message)
         self._stop()
 
+      # Update metrics.
+      # TODO(xujyan): This doesn't rule out duplicates, etc. We can consider updating these metrics
+      # in the launcher.
+      if status.state == mesos_pb2.TASK_FINISHED:
```

Comment:
You mean 

`updates_lost`, `updates_finished`, etc.? 

Read a bit strange though. These are indeed updates about tasks. 

I think the past tense captured what they are.

They are *tasks that have failed* which indicates a monotonically increasing counter, as oppose to `tasks_running`: tasks that are current running, which belongs to something akin to Mesos `state.json`.

If it is about the TODO I am putting up there, I think it would be nice to have duplicated removed even if we call them `updates` so that fact itself doesn't seem to pertain to tasks vs. updates.

What do you think?